src/custom/pages/App/index.tsx

```diff
@@ -92,6 +94,7 @@ export default function App() {
     <>
       <RedirectAnySwapAffectedUsers />
       <Wrapper>
+        <SideBanner type={'anniversary'} />
```

Comment:
can we load this as well with suspense and include in the body?

Reply:
Yes, done in the other PR now.